.gitignore

```diff
@@ -8,6 +8,9 @@
 # Emacs
 *~
 
+# IntelliJ Idea-based IDEs
+.idea/
```

Comment:
Every developer gets to make their political statement in .gitignore. Mine is on line 8 :)